maven-plugin/src/main/java/io/confluent/kafka/schemaregistry/maven/SchemaRegistryMojo.java

```diff
@@ -49,6 +49,9 @@
   @Parameter(required = false)
   List<String> schemaProviders = new ArrayList<>();
 
+  @Parameter
+  Map<String, String> sslConfigs = new HashMap<>();
```

Comment:
Maybe change `sslConfigs` to `configs`, since any valid config can be specified.  wdyt?

Reply:
Sgtm